src/net/chat.rs

```diff
@@ -1792,6 +1883,42 @@ impl ConversationP2P {
         Ok(None)
     }
 
+    /// Validate a pushed stackerdb chunk
+    /// Update bandwidth accounting, but forward the stackerdb chunk along.
+    /// Possibly return a reply handle for a NACK if we throttle the remote sender
```

Comment:
How is a pushed chunk validated? I think the rustdoc should have just a simple explanation of what is being checked here.

Also, what does forward the "stackerdb chunk along" mean? Is this method not only validating, but also forwarding the message to other peers?

Reply:
This is only verifying that the message is well-formed, as happens in all the other similarly-named methods for transactions, blocks, and microblocks.  The "forward along" comment refers to forwarding it back up to the reply handle that the (in this case) stackerdb state machine is using to retrieve the message.  Contrast this to messages that don't need to be forwarded, because no one is expecting them.